File: comms/bulk-messaging/sender-pools/fetch-sender-pool-operation/fetch-sender-pool-operation.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.sender_pools.fetch_operation(
    operation_id="comms_operation_01h9krwprkeee8fzqspvwy6nq8",
)

File: comms/bulk-messaging/sender-pools/fetch-sender-pool-operation/fetch-sender-pool-operation.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.sender_pools.fetch_operation(operation_id: "comms_operation_01h9krwprkeee8fzqspvwy6nq8")
